frontend/public/components/job.tsx

```diff
@@ -145,8 +145,7 @@ const JobTableRow = ({
 JobTableRow.displayName = 'JobTableRow';
 
 const jobStatus = (job: JobKind): string => {
-  const conditions = _.get(job, 'status.conditions', null);
-  return conditions && conditions.length > 0 ? conditions[0].type : 'In Progress';
+  return job && job.status ? _.get(job, 'status.conditions[0].type', 'In Progress') : null;
```

Comment:
I've left this change because I think it's a little better as long as we're using `_.get`